toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -187,6 +193,25 @@ func systemdFormatPartitionId(idType imagecustomizerapi.IdType, id string) (stri
 	}
 }
 
+func systemdFormatCorruptionOption(corruptionOption *imagecustomizerapi.CorruptionOption) (string, error) {
+	if corruptionOption == nil {
+		return "", nil
+	}
+
+	switch *corruptionOption {
+	case imagecustomizerapi.CorruptionOptionDefault:
+		return "", nil
```

Comment:
Should the default value be mapped to `"panic-on-corruption"`?

Reply:
No, the empty option will make the corruption logged and fail the booting with I/O errors. The panic not only will print out / log the error, but also will try rebooting the image..

Reply:
Oh. We should give all the options a proper name. And the default option (`""`) should be equivalent to one of the named options.

Perhaps name it `CorruptionOptionIoError` and `"io-error"`?

Reply:
Make sense, fixed, please check.